### PR TITLE
Content has a max-width of 1000px

### PR DESCRIPTION
**Pull request by Mariam-mohamed110**
Contents have a max-width of 1000px
On screens smaller than 1000px, the app behaves as before
A few minor styling changes for phones for consistency

### CODE REVIEW

**Review by yuna-iwata — CHANGES_REQUESTED** [review]
should the header logo and buttons max width also be set to 1000px?  and also the margins seem to have disappeared between 1000px and phone width
Screenshot 2023-03-14 at 13 57 54

File: web/src/Styling/TopicBreadcrumbs/TopicBreadcrumbs.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@
   @include respond-to("large") {
     display: block;
     margin-top: 30px;
-    margin-left: 14.6%;
+    // margin-left: 14.6%;
```
Comment (s-dilip) [inline web/src/Styling/TopicBreadcrumbs/TopicBreadcrumbs.scss:15]: Can probably get rid of this commented line of code if no longer needed.
```suggestion
 
```

File: web/src/Styling/TopicCard/TopicCard.scss [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 .topic-card-parent {
   @include screen-size-change;
-  margin: 0px;
-  margin-top: 20px;
+  // margin: 0px;
+  // margin-top: 20px;
```
Comment (s-dilip) [inline web/src/Styling/TopicCard/TopicCard.scss:6]: Could remove unneeded code.
```suggestion

```

**Comment by DomVinyard** [comment]
> should the header logo and buttons max width also be set to 1000px? and also the margins seem to have disappeared between 1000px and phone width
> Screenshot 2023-03-14 at 13 57 54

Yes, well spotted - usually we want the same inner element to apply to both the header and the content: 

```
<style>
inner {
    max-width: 1000px;
    margin: 0 auto;
}
</style>

<div className="header-container">
   <div className="inner">
       I am a header
   </div>
<div>
<div className="content-container">
   <div className="inner">
       I am the main container
   </div>
<div>
```

this way we can give the header and the content different background colours that are full width, but the inner element is always consistent

**Comment by Mariam-mohamed110** [comment]
> Under `1000px` width, there is no `margin-left` for content (Breadcrumbs, Topic Card, Relation Cards)whatsoever which may not be ideal.
> 
> image

**Review by yuna-iwata — APPROVED** [review]
looks v good 😊

**Review by s-dilip — APPROVED** [review]

File: web/src/Global/Header/Header.css [hunk 3]
```diff
@@ -1,83 +1,56 @@
...
 }
-
-header a {
-  display: -webkit-box;
-  display: -ms-flexbox;
+header .inner {
+  display: flex;
+  justify-content: space-between;
+  height: 70px;
+  max-width: 1000px;
```
Comment (Ademsk1) [inline web/src/Global/Header/Header.css:22]: Very cool !

**Review by Ademsk1 — APPROVED** [review]
Looks good!

**Pull request merged**